ci/install-dracut-sshd.sh

```diff
@@ -51,7 +51,7 @@ else
     sed -i 's/^\(GRUB_CMDLINE_LINUX=\)"\([^"]\+\)"/\1"\2 rd.neednet=1 ip=dhcp"/' /etc/default/grub
     grub2-mkconfig -o  /etc/grub2.cfg
 fi
-dracut -f -v
+dracut -f -v --install-optional /etc/systemd/network/20-wired.network
```

Comment:
I think it would be more natural to follow the example in the README, you've added.

Then, analogously to how the module is installed

```
scp -r -P $port "${ssh_flags[@]}" \
    "$base"/46sshd root@"$host":/usr/lib/dracut/modules.d
```

just copy the dracut.conf.d drop-in, e.g.:

```
scp -r -P $port "${ssh_flags[@]}" \
    "$base"/example/90-networkd.conf root@"$host":/etc/dracut.conf.d
```

Reply:
With that the `dracut -f -v` call could remain as is.